src/Illuminate/Routing/Router.php

```diff
@@ -360,6 +361,21 @@ public function resource($name, $controller, array $options = [])
     }
 
     /**
+     * Register a new redirect route with the router.
+     *
+     * @param  string  $from
+     * @param  string  $to
+     * @param  int  $status
+     * @return \Illuminate\Routing\Route
+     */
+    public function redirect($from, $to, $status = 301)
+    {
+        return $this->get($from, function () use ($to, $status) {
```

Comment:
This should probably be a `$this->any()` call as you could be doing a 303/307 redirect from a POST/PUT.